source/AndroidXProject.cshtml

```diff
@@ -44,10 +44,10 @@
   }
   <PropertyGroup>
     <PackageId>@(Model.NuGetPackageId)</PackageId>
-    <Title>Xamarin AndroidX - @(Model.Name)</Title>
-    <Summary>Xamarin.Android bindings for AndroidX - @(Model.Name)</Summary>
-    <Description>Xamarin.Android bindings for AndroidX - @(Model.Name)  artifact=@(Model.MavenGroupId):@(Model.Name) artifact_versioned=@(Model.MavenGroupId):@(Model.Name):@(artifact_version)</Description>
-    <PackageTags>Xamarin AndroidX Xamarin.AndroidX Support Google @(Model.Name) artifact=@(Model.MavenGroupId):@(Model.Name) artifact_versioned=@(Model.MavenGroupId):@(Model.Name):@(artifact_version)</PackageTags>
+    <Title>.NET Android (formerly Xamarin.Android) AndroidX - @(Model.Name)</Title>
```

Comment:
Other metadata updates use `.NET for Android` instead of `.NET Android`. We *think* (😆) our product name is `.NET for Android` so let's standardize on that.

Reply:
Yes I was more for ".NET for Android",  but I came across ".NET Android" too. Will change